Jenkinsfile

```diff
@@ -114,6 +114,10 @@ rocmtest clang_debug: rocmnode('cdna') { cmake_build ->
         cmake_build(flags: "-DCMAKE_BUILD_TYPE=release")
         stash includes: 'build/*.deb', name: 'migraphx-package'
     }
+}, hidden_symbols: rocmnode('cdna') { cmake_build ->
+    stage('Hidden symbols') {
+        cmake_build(flags: "-DMIGRAPHX_ENABLE_PYTHON=Off -DMIGRAPHX_ENABLE_GPU=On -DMIGRAPHX_ENABLE_CPU=On -DCMAKE_CXX_VISIBILITY_PRESET=hidden -DCMAKE_C_VISIBILITY_PRESET=hidden")
```

Comment:
enable FPGA as well ?

Reply:
FPGA doesnt have export macros. I dont think it was added with windows support yet since its incomplete.